app/services/idv/steps/threat_metrix_step_helper.rb

```diff
@@ -63,6 +73,7 @@ def log_irs_tmx_fraud_check_event(result)
 
         irs_attempts_api_tracker.idv_tmx_fraud_check(
           success: success,
+          fraud_event_id: fraud_event.id,
```

Comment:
I am not sure we need this.

I naively started to pass in IRS and Login session IDs here, but they have that in the normal event payload here, since this comes from an interactive session.